server/service/software.go

```diff
@@ -105,6 +105,11 @@ func (svc *Service) ListSoftware(ctx context.Context, opt fleet.SoftwareListOpti
 		return nil, nil, err
 	}
 
+	// Vulnerability filters are only available in premium
+	if !opt.IncludeCVEScores && (opt.MaximumCVSS > 0 || opt.MinimumCVSS > 0 || opt.KnownExploit) {
```

Comment:
Recommend adding a comment that `IncludeCVEScores` is always set for Premium.

Reply:
added